discovery/kubernetes/kubernetes.go

```diff
@@ -258,103 +267,138 @@ const resyncPeriod = 10 * time.Minute
 // Run implements the discoverer interface.
 func (d *Discovery) Run(ctx context.Context, ch chan<- []*targetgroup.Group) {
 	d.Lock()
-	namespaces := d.getNamespaces()
+	var namespaces []string
+	if d.memberRollNamespace != "" && d.memberRollController != nil {
+		namespaces = []string{d.memberRollNamespace}
+	} else {
+		namespaces = d.getNamespaces()
```

Comment:
Do we still need this?

Reply:
The code does if the member roll information is not provided